core/src/main/java/org/infinispan/remoting/rpc/ResponseMode.java

```diff
@@ -11,6 +11,11 @@
  */
 public enum ResponseMode {
    SYNCHRONOUS,
+   /**
+    * Use this when doing a broadcast to all members to prevent SuspectExceptions from nodes that
+    * are leaving - these can be still part of the view but it is not part of the topology and
+    * the cache is not running there.
```

Comment:
This comment is a bit too narrow. First, we use `SYNCHRONOUS_IGNORE_LEAVERS` for anycast RPCs as well, not just for broadcasts. Second, we still support asymmetric clusters, and there the nodes that don't have your cache running will always return a `CacheNotFoundResponse`, which translates into a `SuspectException` with `SYNCHRONOUS`.